array-manipulation-1/src/last.js

```diff
@@ -0,0 +1,9 @@
+// /* exported last */
+function last(array) {
+  var arrayLast = 1;
+  for (var i = array.length - 1; i >= array.length - 1; i--) {
+    arrayLast = array[i];
+  }
+  return arrayLast;
```

Comment:
You don't actually need a loop here! You can just return the last index of the array like so 
```
function last(array) {
  var output = array[array.length - 1];
  return output;
}
```